core/shared/utils/bh_atomic.h

```diff
@@ -44,6 +46,9 @@ extern "C" {
  * #endif
  */
 
+typedef uint64_t uint64;
+typedef uint32_t uint32;
+typedef uint16_t uint16;
```

Comment:
If including `bh_platform.h`, we don't need to add the three lines, since these types are already defined in the platform's header files.

Reply:
Removed